<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>53</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <property name="text">
     <string>Ziel:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <property name="text">
     <string>Quelle:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="quelleLineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>591</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Comic Sans MS&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="zielLineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>591</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Comic Sans MS&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="convertButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>401</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 48pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <property name="text">
     <string>Konvertieren</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quelleButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>10</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <property name="text">
     <string>Pfad</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zielButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>50</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <property name="text">
     <string>Pfad</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>zielButton_clicked()</slot>
  <slot>quelleButton_clicked()</slot>
 </slots>
</ui>
